apps/leo_gateway/src/leo_gateway_http_commons.erl

```diff
@@ -1087,13 +1087,21 @@ get_range_object_small(_Req, BucketName, Key, Start, End,
 
 
 %% @private
```

Comment:
Add the comment about why this hack is needed (The description of this PR seems good to me